locale/fr/about/governance.md

```diff
@@ -106,8 +105,8 @@ ou sont des modifications de gouvernance, de politique de contribution, de
 participation au TSC, ou concernant le processus de release.
 
 L'intention du programme n'est pas d'approuver ou relire tous les 
-correctifs. Ceci devra être fait de manière continuelle sur GitHub
-et géré par le groupe des Collaborateurs.
+correctifs. Ceci devra être fait de manière continue sur GitHub
+géré par le groupe des Collaborateurs.
```

Comment:
This sentence sounds weird to me now.

Why removing the `et` part and not applying my suggestion of `par l'ensemble des Collaborateurs`?

It's more in par with the source material I think.

Reply:
Oh my. Sorry, I hadn't seen the last part of your suggestion. Correcting ASAP